package.json

```diff
@@ -81,7 +83,8 @@
     "wc-sass-render": "^1.2.3",
     "web-component-analyzer": "^1.1.6",
     "webpack-merge": "^5.8.0",
-    "whatwg-fetch": "^3.6.2"
+    "whatwg-fetch": "^3.6.2",
+    "sass": "^1.49.9"
```

Comment:
Update to the following 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L74-L78